sdk/runanywhere-kotlin/src/androidMain/kotlin/com/runanywhere/sdk/data/config/ConfigurationLoader.kt

```
@@ -0,0 +1,43 @@
+ * Loads configuration files from assets directory
+ */
+internal actual suspend fun loadResourceFile(fileName: String): String = withContext(Dispatchers.IO) {
+    try {
+        // Try to get Android context from ServiceContainer
+        // Note: This requires ServiceContainer to have context available
+        val context = try {
+            // Access Android context if available
+            val contextField = ServiceContainer::class.java.getDeclaredField("androidContext")
+            contextField.isAccessible = true
+            contextField.get(ServiceContainer.shared) as? Context
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)